src/app/EventManagement.cpp

```diff
@@ -787,6 +789,27 @@ CHIP_ERROR EventManagement::ScheduleFlushIfNeeded(EventOptions::Type aUrgent)
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR EventManagement::SetScheduledEventEndpoint(EventNumber * apEventEndpoints)
+{
+    CHIP_ERROR err                    = CHIP_NO_ERROR;
```

Comment:
err seems never set. Should we just end this with CHIP_NO_ERROR?

is there a possibility of error here?

Reply:
never error, remove the error, thanks